src/chrome/content/rules/Livefyre.xml

```diff
@@ -44,7 +44,7 @@
 
 
 	<rule from="^http://avatars\.fyre\.co/"
-		to="https://dpstvy7p9whsy.cloudfront.net/" />
+		to="https://avatars.fyre.co/" />
```

Comment:
If you move this to the bottom you may just replace it with `<rule from="^http:" to="https:" />`.